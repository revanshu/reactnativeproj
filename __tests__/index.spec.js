import React from 'react';
import { shallow } from 'enzyme';
import Module from '../src/index';

const navigation = {
  dispatch: jest.fn(),
  openDrawer: jest.fn(),
  state: {
    params: {
      userInfo: {
        countryCode: 'US',
        siteId: 5505,
        userId: 'ajsnow',
      },
    },
  },
};

describe('Button', () => {
  describe('Rendering', () => {
    it('should match to snapshot', () => {
      const component = shallow(<Module navigation={navigation} />);
      expect(component).toMatchSnapshot();
    });
  });
});
