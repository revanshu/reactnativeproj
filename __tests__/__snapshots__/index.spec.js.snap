// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Rendering should match to snapshot 1`] = `
<ContextProvider
  value={
    Object {
      "dispatch": [MockFunction],
      "openDrawer": [MockFunction],
    }
  }
>
  <Provider
    store={
      Object {
        "@@observable": [Function],
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      }
    }
  >
    <NavigationContainer />
  </Provider>
</ContextProvider>
`;
